toonz/sources/toonz/main.cpp

```diff
@@ -385,7 +385,7 @@ int main(int argc, char *argv[]) {
   fmt.setStencil(true);
   QGLFormat::setDefaultFormat(fmt);
 
-#ifdef LINUX
+#if defined(LINUX) || (defined(_WIN32) && defined(__GNUC__))
```

Comment:
I don't know. Official documentation at opengl.org tell that GLUT should be initialized, but MSVC-part of OpenToonz does not call glutInit, i don't know why. I thought that this bug discovered while porting to linux, and the fix was added for LINUX only to avoid potential risks to broke compatibility with windows and mac.

https://www.opengl.org/resources/libraries/glut/spec3/node9.html

Me too don't wanted to broke MSVC- and Mac-code and kept it unchanged. But i think this function should be called at all systems, will be cool if anyone can check this at Mac and/or MSVC.

You right i was should to comment this.